go/test/endtoend/vtgate/schematracker/sharded/st_sharded_test.go

```diff
@@ -86,15 +83,10 @@ func TestMain(m *testing.M) {
 			VSchema:       VSchema,
 			SidecarDBName: sidecarDBName,
 		}
-		clusterInstance.VtGateExtraArgs = []string{"--schema_change_signal",
-			"--vschema_ddl_authorized_users", "%",
-			"--schema_change_signal_user", "userData1"}
-		clusterInstance.VtGatePlannerVersion = planbuilder.Gen4
-		clusterInstance.VtTabletExtraArgs = []string{"--queryserver-config-schema-change-signal",
-			"--queryserver-config-schema-change-signal-interval", "0.1",
-			"--queryserver-config-strict-table-acl",
-			"--queryserver-config-acl-exempt-acl", "userData1",
-			"--table-acl-config", "dummy.json"}
+		clusterInstance.VtGateExtraArgs = append(clusterInstance.VtGateExtraArgs,
+			"--schema_change_signal",
+			"--vschema_ddl_authorized_users", "%")
+		clusterInstance.VtTabletExtraArgs = append(clusterInstance.VtTabletExtraArgs, "--queryserver-config-schema-change-signal")
```

Comment:
Some acl flags have also been dropped. Is that intentional?

Reply:
It is not required anymore. I will add a deprecation note for it.